Gifski/util.swift

```diff
@@ -266,6 +298,37 @@ extension NSAlert {
 		if let informativeText = informativeText {
 			self.informativeText = informativeText
 		}
+
+		if let detailText = detailText {
+			if #available(macOS 10.14, *) {
+				let scrollView = NSTextView.scrollableTextView()
+				//scrollView.frame = CGRect(width: 300, height: 120)
+				scrollView.translatesAutoresizingMaskIntoConstraints = false
+				scrollView.heightAnchor.constraint(equalToConstant: 120).isActive = true
+
+				scrollView.onAddedToSuperview {
+					print("added")
+					scrollView.leftAnchor.constraint(equalTo: scrollView.superview!.leftAnchor).isActive = true
+					scrollView.rightAnchor.constraint(equalTo: scrollView.superview!.rightAnchor).isActive = true
```

Comment:
I could use some help with the auto-layout constraints. I want to make the scroll view take up as much width as available without making the alert panel wider. The above just makes it 0 width...

// @sunshinejr

Reply:
Yeah _I think_ that the `scrollView.superview` doesn't really have a width and because of that the `scrollView` won't have as well (so the wrapper on top of `accessoryView` is growing based on `accessoryView`, not the other way around). You might have to add a `width` constraint to the `scrollView`, but not sure what should we based that on. There are two `NSTextField`s but this seems really hacky to base it on. Wonder if going with a frame "just-enought" wouldn't be okay for now.

Reply:
I tried that (https://github.com/sindresorhus/gifski-app/pull/84/commits/382946db9773d5f402743b968f0737de0d5de99a), but it still doesn't work...

I have a solution for doing it manually (commented out), but I would really prefer using auto-layout for this.

Reply:
I'm giving up on auto-layout for this...